--- a/shellinabox/vt100.jspp
+++ b/shellinabox/vt100.jspp
@@ -300,7 +300,8 @@ VT100.prototype.getUserSettings = functi
 
   // Enable soft keyboard icon on some clients by default.
   if (navigator.userAgent.match(/iPad|iPhone|iPod/i) != null ||
-      navigator.userAgent.match(/PlayStation Vita|Kindle/i) != null) {
+      navigator.userAgent.match(/PlayStation Vita|Kindle/i) != null ||
+      navigator.userAgent.match(/Mobile|Tablet/i) != null) {
     this.softKeyboard       = true;
   }
 
